src/script/pages/app-basepack.ts

```diff
@@ -362,15 +362,14 @@ export class AppBasePack extends LitElement {
                 <h5>Up next</h5>
 
                 <p>
-                  After uploading the above files to your Web App tap Run New Test to test your PWA again, or tap Next to go straight to packaging! 
+                  After uploading the above files to your Web App tap Run New Test to test your PWA again!
```

Comment:
maybe place this string in the localized module?

Reply:
ahh yeah, did that with other strings this morning but it didnt click in my head to do it for this one. Ill do it before merging